part0.md

```diff
@@ -0,0 +1,24 @@
+Percentile Exercises
+
+Part 0 - Statistics
+
+Read https://en.wikipedia.org/wiki/Percentile through the section on the  “The nearest-rank method” and https://en.wikipedia.org/wiki/Quantile through “Examples”
+
+0.1. Given 1000 measures: How many are above the 90th percentile?  How many are below the 90th percentile?  Above and below the 99.5th percentile?
+
+    - 900 are below, 100 are above
+    - 995 are below, 5 are above
+
+
+
+0.2 How many measures does a dataset need to have the 99.999th percentile be distinct from the 100th percentile (max) of a dataset?
+
+    - 100,000
+0.3 Given a large unknown dataset, what is the probability that one randomly selected measure exceeds the 99th percentile? What is the probability that at least one measure in a sample of 50 that exceeds the 99th percentile of the dataset?
+
+    - 0.5%
+    - 1%
+
+0.4 Given a large unknown dataset, how many measures are needed to have a .5 probability of having 1 or more measures exceed the 99.9th percentile?
+
+    - 1,000
```

Comment:
This is not correct either.  Here we take the math from 0.3: `P = 1 - .99^x`.  We know P = .5, so `.5 = 1 - .99^x` and solve for x. Hint, the `log` function is the inverse of 'to the power of'  You'll need log base .99 to solve the equation.  Wolfram alpha can help you here.